### PR TITLE
pdfbox: Update to version 2.0.25

### PR DESCRIPTION
**Pull request by sualeh**

### CODE REVIEW

**Review by tech189 — APPROVED** [review]

**Comment by sualeh** [comment]
@rashil2000  - are you able to merge this in? @tech189 has taken a look. Thanks.

**Comment by tech189** [comment]
Could you add autoupdate so that we don't need to manually update this manifest in future? You could base it on my suggestion https://github.com/ScoopInstaller/Versions/pull/358#pullrequestreview-840599804 where I added autoupdate.

**Comment by sualeh** [comment]
@tech189  - thanks for your gentle tutorial. :-)

**Review by tech189 — APPROVED** [review]

**Comment by tech189** [comment]
No problem, it all looks good now!

**Comment by sualeh** [comment]
@rashil2000 - this one is ready to go! Thanks for all your help, @tech189.

File: bucket/pdfbox.json [hunk 1]
```diff
@@ -1,22 +1,38 @@
...
+    "checkver": "Apache PDFBox ([\\d.]+) released",
+    "autoupdate": {
+      "url": [
+        "https://downloads.apache.org/pdfbox/$version/pdfbox-app-$version.jar#/pdfbox.jar",
+        "https://downloads.apache.org/pdfbox/$version/preflight-$version.jar#/preflight.jar"
+      ],
+      "hash": [
+        {
+          "url": "$url.sha512",
+          "regex": "$sha512"
```
Comment (rashil2000) [inline bucket/pdfbox.json:30]:
```suggestion
          "url": "$url.sha512"
```

File: bucket/pdfbox.json [hunk 2]
```diff
@@ -1,22 +1,38 @@
...
+        },
+        {
+          "url": "$url.sha512",
+          "regex": "$sha512"
```
Comment (rashil2000) [inline bucket/pdfbox.json:34]:
```suggestion
          "url": "$url.sha512"
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Comment by niheaven** [comment]
Only need one hash section

**Pull request merged**